assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py

```diff
@@ -401,7 +424,35 @@ def finetune(args: Namespace):
             f"Forcing `mlflow_hftransformers_misc_conf` to set to {mlflow_hftransformers_misc_conf} "
             f"for {model_name_or_type}"
         )
-        setattr(args, "mlflow_hftransformers_misc_conf", mlflow_hftransformers_misc_conf)
+        args.mlflow_ft_conf["mlflow_hftransformers_misc_conf"] = mlflow_hftransformers_misc_conf
+
+    # if MLmodel file exists pass to finetuned model as `base_model_mlmodel`
+    mlflow_config_file = Path(args.model_selector_output, MLFlowHFFlavourConstants.MISC_CONFIG_FILE)
+    if mlflow_config_file.is_file():
+        import yaml
+        mlflow_data = None
+        try:
+            with open(mlflow_config_file, "r") as rptr:
+                mlflow_data = yaml.safe_load(rptr)
+        except Exception as e:
+            logger.info(f"Unable to load MLmodel file - {e}")
+        if mlflow_data is not None:
+            # pass base model MLmodel file data if available
+            mlflow_hftransformers_misc_conf = args.mlflow_ft_conf.get("mlflow_hftransformers_misc_conf", {})
+            mlflow_hftransformers_misc_conf.update({"base_model_mlmodel": mlflow_data})
+            args.mlflow_ft_conf["mlflow_hftransformers_misc_conf"] = mlflow_hftransformers_misc_conf
+            logger.info(f"Setting `base_model_mlmodel` in finetuned mlflow model - {mlflow_hftransformers_misc_conf}")
+
+            # pass base model signature if available
+            mlflow_model_signature = mlflow_data.get("signature", {})
+            args.mlflow_ft_conf["mlflow_model_signature"].update(mlflow_model_signature)
+            logger.info(f"Updating signature from base model - {mlflow_model_signature}")
+        else:
+            logger.info("MLmodel file is empty")
+    else:
+        logger.info("MLmodel file does not exist")
+
+    logger.info(f"FT MLFlow config - {args.mlflow_ft_conf}")
```

Comment:
remove this as base model signatures are for fill mask task

Reply:
removed the dependency on the base model signature